pgxn/neon/Makefile

```diff
@@ -11,10 +11,11 @@ OBJS = \
 	pagestore_smgr.o \
 	relsize_cache.o \
 	walproposer.o \
-	walproposer_utils.o
+	walproposer_utils.o \
+	console_connector.o
 
 PG_CPPFLAGS = -I$(libpq_srcdir)
-SHLIB_LINK_INTERNAL = $(libpq)
+SHLIB_LINK_INTERNAL = $(libpq) -lcurl
```

Comment:
I wasn't sure what the proper procedure for adding dependencies to extension is, but this worked on my laptop for now

Reply:
I think this would be the proper way:
```
SHLIB_LINK = -lcurl
````

Reply:
[`SHLIB_LINK` is for external libraries, `SHLIB_LINK_INTERNAL` is for libraries that are in the Postgres source tree.](https://github.com/postgres/postgres/blob/637dce2254245321283ade9db1b7cc8d1f8cf308/src/Makefile.shlib#L30). (Not sure why in-tree and out-of-tree libraries need to be treated separately)

Reply:
Thanks, switched it